templates/README.md

```diff
@@ -4,20 +4,18 @@
 
 Jobs and Prow configuration are generated from templates. The `templates` directory has the following structure:
 
+- `data` which is the subdirectory with files that describes jobs that the [Render Templates](../development/tools/cmd/rendertemplates) tool should generate using job definitions from templates.
```

Comment:
```suggestion
- `data`, which is the subdirectory with files that describe jobs that the [Render Templates](../development/tools/cmd/rendertemplates) tool should generate using job definitions from templates.
```

I assume the files describe jobs, not the subdirectory describes jobs?

Reply:
yes